Document use of "chef-client -j file.json" passed attributes (#2201)
chef/chef-web-docs: chef_master/source/attributes.rst

* Many attributes are maintained in the chef-repo for Policyfiles, environments, roles, and cookbooks (attribute files and recipes)
* Many attributes are collected by Ohai on each individual node at the start of every Chef Infra Client run
* The attributes that are maintained in the chef-repo are uploaded to the Chef Infra Server from the workstation, periodically

> * Chef Infra Client will pull down the node object from the Chef Infra Server (which contains the attribute data from the previous Chef Infra Client run, including those set via ``-j`` with JSON files), after which all attributes (except ``normal`` are reset)

Reviewer: ```suggestion
* Chef Infra Client will pull down the node object from the Chef Infra Server and then reset all the attributes except ``normal``. The node object will contain the attribute data from the previous Chef Infra Client run including attributes set with JSON files via ``-j``.
```
